JavaFXUserInterface/src/main/java/SystemsController.java

```diff
@@ -276,4 +320,33 @@ public void generateList() {
         systemsListPane.getChildren().addAll(table);
 
     }
+
+    public ObservableList<Asset> sortSystems(String selectedSort) {
+        //Copying the systems Assets list into another ObservableList so as to not impact the original one.
+        ObservableList<Asset> sortedSystems = FXCollections.observableArrayList(systems);
+        ObservableList<Asset> descending = FXCollections.observableArrayList(ModelController.getInstance().getAllLiveAssets("Descending RUL").subList(0,10));
```

Comment:
don't make a call to the db fo no reason, call it only if you need it, this is the kind of stuff that slows down the application